### PR TITLE
Friday drink - Annika Melin

### PR DESCRIPTION
**Pull request by ankimelin**
Link to my deployed project:
https://wizardly-noyce-4f8da0.netlify.app/

My reflections on how this week's project turned out:
A fun and light styling, relaxing to do after past few weeks of portfolio and music releases (not sure I can keep it going forward when looking for work though?). I'm happy that I split the components and made that work - it was the challenge of the week. Also happy I created my own data file (drinks), which I so far use in Result component (next up Questions component) - thanks Van for the help!

Things I'd like to have clarified or explained in more detail:
Would have been nice to spend more than 10 last min of Wed session on passing state values between components!

### CODE REVIEW

File: code/src/css/button.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  font-family: 'Gochi Hand', cursive;
+  font-size: 24px;
+  color: purple;
+  border: 4px dotted purple;
+}
+
+button:hover {
+  cursor:url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg'  width='40' height='48' viewport='0 0 100 100' style='fill:black;font-size:24px;'><text y='50%'>🍹</text></svg>") 16 0,auto; /*!emojicursor.app*/
+}
+
```
Comment (ssundblom) [inline code/src/css/button.css:17]: This.Is.Amazing! Love this 'little' detail! Will for sure take this with me to our next project!

File: code/src/js/Drinks.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+export const drinks = [
+  {
+    id: "aperol",
+    name: "Aperol Spritz",
+    img: "https://images.unsplash.com/photo-1585386116149-22ad0453f631?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60"
+  },
```
Comment (ssundblom) [inline code/src/js/Drinks.js:6]: Very clever to do it this way! Very clean and nice way to display info.

File: code/src/js/Form.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  const [alcohol, setAlcohol] = useState("Conventional")
+  const handleAlcoholChange = newAlcohol => {
+    setAlcohol(newAlcohol)
+  }
+
+  const [result, setResult] = useState(false)
+  const handleSubmit = event => {
+    event.preventDefault()
+    setResult(true)
+  }
```
Comment (ssundblom) [inline code/src/js/Form.js:30]: This is so nice! You're very good at naming your variebles! Makes it easier to understand 👍

File: code/src/js/Header.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      <header>
+        <img className="emoji" src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/microsoft/209/glowing-star_1f31f.png" alt="glowing star"></img><h1> Demo <span className="fun">Friyay</span>: it's drink o'clock! </h1><img className="emoji" src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/microsoft/209/tropical-drink_1f379.png" alt="tropical drink"></img>
+      </header>
+    </>
+  )
+}
+
+//glowing star: 🌟 https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/microsoft/209/glowing-star_1f31f.png
+
+//tropical drink: 🍹 &#127865 https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/microsoft/209/tropical-drink_1f379.png
```
Comment (ssundblom) [inline code/src/js/Header.js:17]: Maybe delete the out commented code? :)

File: README.md [hunk 5]
```diff
@@ -1,13 +1,11 @@
...
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+A form taking in input displayed to the user when clicking submit.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+First time using state in React! Started with creating a form and style it, thereafter approached the part of connecting the state values to the user summary.
```
Comment (ssundblom) [inline README.md:8]: i had the same approach! What do you think, was it the best way to start of the project?

File: code/src/css/button.css [hunk 6]
```diff
@@ -0,0 +1,17 @@
+button {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding: 10px;
+  border-radius: 40px;
+  background-color: lightgreen;
+  font-weight: 700;
+  font-family: 'Gochi Hand', cursive;
+  font-size: 24px;
+  color: purple;
```
Comment (ssundblom) [inline code/src/css/button.css:10]: Super super small detail, but I saw that you mentioned font-family further down as well. Maybe put them in body so you don't need to repeat? (but if you only mention it twice maybe there's no need for that?)

**Review by ssundblom — APPROVED** [review]
Hey Annika! Good job! I don't feel that I have anything to comment really. You're an expert at at naming variables and it feels like you're functions are so thought through witch makes the code super easy to read AND to understand. 

It is a very funny topic you've chose! And the styling fits perfectly with it! You're project meets the requests of blue level! Weell done <3

File: code/src/js/Result.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+            <img src={drinkOrder.img} alt="drinkObject.name"></img>
+            <figcaption><span className="fun">{alcohol}</span><br></br> {drinkOrder.name}</figcaption>
+          </figure>
+
+        </>
+      ) : (
+          <figure className="card">
+            <img src="https://images.unsplash.com/photo-1564644411757-a723deba07a8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60" alt="Water"></img>
+            <figcaption><span className="fun">Glass of</span><br></br> Water</figcaption>
+          </figure>
```
Comment (HIPPIEKICK) [inline code/src/js/Result.js:26]: 😆 ⭐

**Pull request closed without merging**